MODULE.bazel

```diff
@@ -30,10 +30,9 @@ register_toolchains("@ruby_toolchains//:all")
 #
 # Rules Python
 #
-
-# aspect_rules_py is promising, but broken on 3.11+ and its `py_venv`
+# aspect_rules_py is promising, but its `py_venv` is 3.11+ and
 # rule is either underdocumented or broken.
```

Comment:
I think it's setup to make a venv per `py_binary`

Reply:
why lol